java-benchmarks/mx.java-benchmarks/mx_java_benchmarks.py

```diff
@@ -1924,6 +1924,14 @@ def postprocessRunArgs(self, benchname, runArgs):
             else:
                 return ["-r", str(iterations)] + remaining
 
+    def vmArgs(self, bmSuiteArgs):
+        vm_args = super(RenaissanceBenchmarkSuite, self).vmArgs(bmSuiteArgs)
+        if java_home_jdk().javaCompliance > '16':
```

Comment:
I think this should go to the image builder? @ivan-ristovic can help you with that.